app/src/main/res/values/dimens.xml

```diff
@@ -1,11 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="dp_55">55dp</dimen>
+    <dimen name="dp_3">3dp</dimen>
+    <dimen name="dp_200">200dp</dimen>
+    <dimen name="dp_60">60dp</dimen>
+    <dimen name="dp_275">275dp</dimen>
+    <dimen name="dp_40">40dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_40` appears to be unused
 Unused resources make applications larger and slow down builds.